src/routes/LandingPage/LandingPage.connector.js

```diff
@@ -0,0 +1,37 @@
+import { connect } from 'react-redux'
+import { push } from 'connected-react-router'
+import { communityUrl } from 'util/navigation'
+import fetchCommunity from 'store/actions/fetchCommunityBySlug'
+import getCommunityForCurrentRoute from 'store/selectors/getCommunityForCurrentRoute'
+import getRouteParam from 'store/selectors/getRouteParam'
+import presentCommunity from 'store/presenters/presentCommunity'
+
+export function mapStateToProps (state, props) {
+  let community, communityTopic
+  const communitySlug = getRouteParam('slug', state, props)
+
+  if (communitySlug) {
+    community = presentCommunity(getCommunityForCurrentRoute(state, props))
+    // communityTopic = getCommunityTopicForCurrentRoute(state, props)
```

Comment:
remove comments

Reply:
or are you still figuring out how to do this?

Reply:
This is still a WIP - will remove comments once it's stable